homeassistant/components/melcloud/climate.py

```diff
@@ -73,6 +83,18 @@
         True,
     )
 
+    platform = entity_platform.current_platform.get()
+    platform.async_register_entity_service(
+        "set_vane_horizontal",
```

Comment:
Please create a constant for this service e.g. `SERVICE_SET_VANE_HORIZONTAL`

Reply:
Done